app/helpers/crayons_helper.rb

```diff
@@ -29,8 +29,8 @@ def crayons_icon_tag(name, native: false, **opts)
       opts.delete(:class),
     ].compact.join(" ")
 
-    opts[:width] ||= 24
-    opts[:height] ||= 24
+    opts[:width] ||= 32
+    opts[:height] ||= 32
```

Comment:
This is global setting and gets applied to all the crayons icon on the web, as a result we see following changes

### Result on this branch
<img width="385" alt="Screenshot 2023-02-09 at 2 36 00 PM" src="https://user-images.githubusercontent.com/9396084/217769641-ce5d9863-7d15-47d4-b675-7cbff4453035.png">

### Result on dev.to
<img width="270" alt="Screenshot 2023-02-09 at 2 36 11 PM" src="https://user-images.githubusercontent.com/9396084/217769588-c9ae772e-a21c-48fa-8bdf-b57d3a3de672.png">

Reply:
In `app/views/articles/_multireaction_button.html.erb` we make following change
Before: `<%= image_tag image_active_path, aria_hidden: true, height: 24, width: 24 %>`
After: `<%= image_tag image_active_path, aria_hidden: true, height: 32, width: 32 %>`

This should fix the icon size.